db/seeds.rb

```diff
@@ -8,15 +8,45 @@
 User.create(username: "test", email: "test@m.com", password: "12345678")
 
 Title.create([
-  { value: "AG.LND.FRST.ZS", name: "Forest area (% of land area)" },
-  { value: "EG.USE.PCAP.KG.OE", name: "Energy use (kg of oil equivalent per capita)" },
+  { value: "SP.POP.TOTL", name: "Population, total"},
```

Comment:
Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.<br>Space inside } missing.